### DISCUSSION TITLE
阿里云盘显示：failed get objs: failed to list objs: not found device info

### DISCUSSION THREAD

**Discussion by MaXin-noob** [AlistGo/alist#3524] [Q&A] [answered]
今天登录的时候突然发现阿里云的驱动失效了，具体表现如下：
![](https://download.xinmouren.cn/2023/02/19/63f239e50f866.png)
在此之前是好用的。
我重新配置了“刷新令牌”依旧无法解决。

**Comment by MaXin-noob** [comment]
更新了Alist发现能用了，想问一下Alist有没有能够提示目前是否为最新版本的功能？

**Comment by qwerttvv** [comment]
一样的问题

failed get objs: failed to list objs: not found device info

配置什么的哪里都没有改，挂载分享链接是好的，挂载我自己账号下目录就这样，版本是最新的
至少在上周1还是好的。

**Comment by HUANMVP** [comment, marked as answer]
阿里云盘更新了，第三方用不了，等大佬们更新看看有没有解决办法

**Reply by yimouleng** [reply]
期待解决

**Comment by iamyaofeng** [comment]
已经更新最新版，还是报错？怎么办呢？

**Reply by iamyaofeng** [reply]
知道原因了，是官方接口改了，删除原来的，新增选阿里云open驱动，具体配置步骤看文档说明，我已经解决了。

**Reply by zhoutao-cn** [reply]
可是，改用 OPEN 之后只解决了用户本人空间的连接问题；挂载他人的分享资源速度非常慢。
有没有解决方案？
